.github/workflows/linux_release.yml

```diff
@@ -49,9 +51,11 @@ jobs:
       working-directory: ./build
       shell: bash
       run: |
-        cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
-              -DLINK_WITH_PYTHON=ON -DBUILD_FORTRAN=ON -DBUILD_PACKAGE:BOOL=ON \
-              -DDOCUMENTATION_BUILD="BuildWithAll" -DTEX_INTERACTION="batchmode" -DENABLE_PCH=OFF \
+        cmake -DCMAKE_BUILD_TYPE:STRING=$BUILD_TYPE \
+              -DLINK_WITH_PYTHON:BOOL=ON -DPYTHON_VERSION:STRING=${{ steps.setup-python.outputs.python-version }} \
+              -DPython_ROOT_DIR=$RUNNER_TOOL_CACHE/Python/${{ steps.setup-python.outputs.python-version }}/x64/ \
```

Comment:
Pass the PYTHON_VERSION cmake variable to make cmake request an EXACT match. We request the **actual** version that was picked by actions/setup-python. So we request 3.8 for eg, and we get 3.8.12 from setup-python and use 3.8.12 here